control-plane/rest/service/src/v0/states.rs

```diff
@@ -0,0 +1,12 @@
+use super::*;
+use common_lib::types::v0::message_bus::{GetStates, States};
+use mbus_api::message_bus::v0::{MessageBus, MessageBusTrait};
+
+pub(super) fn configure(cfg: &mut actix_web::web::ServiceConfig) {
+    cfg.service(get_states);
+}
+
+#[get("/states")]
+async fn get_states() -> Result<Json<States>, RestError> {
```

Comment:
Is there any benefit on exposing these through rest? We already get the states by querying the specific resource.

Reply:
I thought it might be useful to have this. I will leave this as is for now and will review it as part of the next set of changes.